LibreNMS/Billing.php

```diff
@@ -127,45 +136,51 @@ public static function getLastMeasurement($bill_id): array
 
     private static function get95thagg($bill_id, $datefrom, $dateto): float
     {
-        $mq_sql = 'SELECT count(delta) FROM bill_data WHERE bill_id = ?';
-        $mq_sql .= ' AND timestamp > ? AND timestamp <= ?';
-        $measurements = dbFetchCell($mq_sql, [$bill_id, $datefrom, $dateto]);
-        $measurement_95th = (round($measurements / 100 * 95) - 1);
-
-        $q_95_sql = 'SELECT (delta / period * 8) AS rate FROM bill_data  WHERE bill_id = ?';
-        $q_95_sql .= ' AND timestamp > ? AND timestamp <= ? ORDER BY rate ASC';
-        $a_95th = dbFetchColumn($q_95_sql, [$bill_id, $datefrom, $dateto]);
-        $m_95th = $a_95th[$measurement_95th];
+        $measurements = BillData::query()->where('bill_id', $bill_id)->where('timestamp', '>', $datefrom)->where('timestamp', '<=', $dateto)->count();
```

Comment:
This can be replaced by a withCount()

But since this is a single bill, it won't gain much.

Reply:
That's true.  What do you think about `$measurements = $a_95th->count()` after fetching?